org.unitystation.StationHub.desktop

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env xdg-open
```

Comment:
Ideally should be part of https://github.com/unitystation/stationhub – it's not Flathub specific.